openapi/adintel/adintel.yaml

```diff
@@ -192,6 +287,47 @@ components:
         status: trustedTester
       x-tags:
         - accountStats
+    connectedAccount:
+      title: connectedAccount
+      type: object
+      description: 'An advertising account that’s connected within Advertising Intelligence from integrated campaign management platforms, including Google Adwords, Facebook Ads and Local Ads.'
+      x-examples: {}
+      properties:
+        id:
+          type: string
+          description: The resource ID of the Advertising Account
+        type:
+          type: string
+          description: connectedAccount
+        attributes:
+          type:
+            - object
+          properties:
+            name:
+              type: string
+              description: The name of the ad account
+            currencyCode:
+              type: string
+              description: "\t\nThe currency code that the ad account uses"
+            credentialInvalid:
+              type: boolean
+              description: 'Whether or not if the saved credential for the ad account is invalid, e.g. user no longer have access to the account, or user revoked access.'
+            provider:
```

Comment:
Some extra properties can be added to the `provider` and `currencyCode` fields.  For the provider we should document the three possible enum values, and for currencyCode we should set length limits and specify that it is ISO_4217.  I set these extra properties on the fields of the `accountStats` model, so they should be set the same here.